.gitlab-ci.yml

```diff
@@ -512,6 +532,26 @@ release_image:
     - tags
     - triggers
 
+release_trace-agent:
+  stage: release
+  image: docker:18.06.0
+  dependencies:
+    - deps_deb
```

Comment:
you don't need deps_deb if you override the before scripts with `before_script: []`

Reply:
added